docs/StandardLibraryProgrammersManual.md

```diff
@@ -44,6 +44,20 @@ TODO: Should this subsume or link to [AccessControlInStdlib.rst](https://github.
 1. Frequently Encountered Issues
 
 
+## Coding style
+
+- The stdlib currently has a hard line length limit of 80 characters. 
```

Comment:
Do we actually want to maintain/enshrine this?

Reply:
I hedge it with "currently". This is a contributor guide, so it should document the rules we have. The 80-char limit is definitely something people need to be aware of.

We talked about bumping the limit, and I think we agreed it would be desirable, but we have not decided how/when we do it. I don't think we should change this while we still have an active release branch.